docs/modules/theme/pages/fallback-font.adoc

```diff
@@ -95,3 +95,28 @@ font:
 ----
 
 Now Asciidoctor PDF will look for the emoji in the Symbola font instead of the Noto Emoji font.
+
+== Default Microsoft Windows Font Limitations
+When using the fonts provided by default in Microsoft Windows 10, and possibly other versions, it is important to define a fallback font to provide full coverage of Asciidoctor language features.
+In particular, the https://docs.asciidoctor.org/asciidoc/latest/lists/checklist/[Checklist] feature makes use of the _Ballet Box_ ☐ (U+2610) and _Ballot Box with Check_ "☑" (U+2611) characters.
```

Comment:
Rather than mentioning these two examples, it would be better to point to the following page, which already lists all the characters a custom font is expected to provide. See https://docs.asciidoctor.org/pdf-converter/latest/theme/prepare-custom-font/#required-characters Also, please use a proper xref rather than a qualified URL since the page is within the same site.

Reply:
Implemented in commit 48c098682b30b0afd6feafaa52614244d4f76c0f.